my-app/src/App.css

```diff
@@ -0,0 +1,112 @@
+* {
+  font-family: "Lato", sans-serif;
+  margin: 0;
+  padding: 0;
+  /* font-size: 12px; */
+}
+
+body,
+html {
+  margin: 0;
+  padding: 0;
+}
+
+.post {
+  background: #ffffff;
+  font-family: "Lato", sans-serif;
+  padding: 8px 8px;
+  margin: 0 24px;
+  color: #37474f;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+}
+
+.post-title {
+  color: #37474f;
+  font-size: 16px;
+  line-height: 24px;
+  margin: 0;
```

Comment:
Again, this seems redundant

Reply:
Refactored to remove units from line-height property.